ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+    </item>
+
+  <item xml:id="q19" n="18">
+  <num value="17">፲፯</num>
+  <dim unit="leaf">10</dim>
+    <locus from="162r" to="171v"/>
+   </item>
+
+<item xml:id="q20" n="19">
+  <num value="18">፲፰</num>
+   <dim unit="leaf">10</dim>
+  <locus from="172r" to="181v"/>
+  </item>
+
+<item xml:id="q21" n="20">
+<num value="19">፲፱</num>
+<dim unit="leaf">10</dim>
+ <locus from="182r" to="191v"/>
+  </item>
+
+  <item xml:id="q22" n="21">
+  <num value="20">፳</num>
+  <dim unit="leaf">10</dim>
+   <locus from="192r" to="201v"/>
+    </item>
+
+    <item xml:id="q23" n="22">
+    <dim unit="leaf">2</dim>
+     <locus from="202r" to="203v"/>
+     <note>The last leaf is mulilated.</note>
+      </item>
+  </list>
+  </collation>
+      <condition key="good">
+      </condition>
+  </supportDesc>
+      <layoutDesc>
+        <layout columns="3" writtenLines="27" xml:id="textarea1">
+      <locus from="3r" to="201v"/>
+      <note corresp="#leafdim">Data taken from fol. 17r.</note>
+        </layout>
+
+
+        <layout columns="3" writtenLines="28" xml:id="textarea2">
+            <locus from="202ra" to="203ra"/>
+          <note corresp="#leafdim">Data taken from fol. 202r.</note>
+            </layout>
+             </layoutDesc>
+              </objectDesc>
+
+                <handDesc>
+              <handNote script="Ethiopic" xml:id="h1" corresp="#ms_i1 #ms_i2
+                #ms_i3 #ms_i4 #ms_i5 #ms_i6 #ms_i7 #ms_i8 #a5 #e3">
+                  <date notBefore="1650" notAfter="1750"/>
+                <note>Starting from ca. <locus target="198r"/>
+                  it looks like the scribe had to economize
+                    the space and compress his handwriting, squeezing
+                  a few more letters (up to 16 or 17) into the line. The problem of
+                  space was solved by means of <locus from= "202r" to="202v"> where
+                    the scribe increased the size of his handwriting and the distance
+                  between the letters (ca. 11 to 13 per line).</locus></note>
+                             </handNote>
+
+                      <handNote script="Ethiopic" xml:id="h2" corresp="#a4">
+                          <date notBefore="1772" notAfter="1773"/>
+                        </handNote>
+                      </handDesc>
+
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"/>
+                               <desc type="RecordTransaction"/>
+                                 <q xml:lang="gez">ኍልቈ᎓ እህል᎓ ዘወጽአ᎓ በዘመነ᎓ ሉቃስ᎓></q>
+                                 <note>The note concerns tributes or transaction with cereals.</note>
+                                 </item>
+
+                                  <item xml:id="a2">
+                                    <locus from="1r" to="2r"/>
+                                     <desc type="GuestText" corresp="LIT2872RepCh153">A
```

Comment:
I believe the correct mark up would be `<desc type="GuestText"><title ref="LIT2872RepCh153"/>` (this is in the guidelines), I am not sure whether the way you have done it here will be visualized

Reply:
Done.